test/unit/mrd/test_balanced_strand_utils.py

```diff
@@ -43,201 +42,192 @@ def _assert_files_are_identical(file1, file2):
         assert f1.read() == f2.read()
 
 
-def test_read_balanced_strand_LAv5and6_trimmer_histogram():
-    with tempfile.TemporaryDirectory() as tmpdirname:
-        tmp_out_path = pjoin(tmpdirname, "tmp_out.parquet")
-        df_trimmer_histogram = read_balanced_strand_trimmer_histogram(
-            BalancedStrandAdapterVersions.LA_v5and6,
-            input_histogram_LAv5and6_csv,
-            output_filename=tmp_out_path,
-        )
-        df_trimmer_histogram_from_parquet = pd.read_parquet(tmp_out_path)
-        df_trimmer_histogram_expected = pd.read_parquet(parsed_histogram_LAv5and6_parquet)
+def test_read_balanced_strand_LAv5and6_trimmer_histogram(tmpdir):
+    tmp_out_path = pjoin(tmpdir, "tmp_out.parquet")
+    df_trimmer_histogram = read_balanced_strand_trimmer_histogram(
+        BalancedStrandAdapterVersions.LA_v5and6,
+        input_histogram_LAv5and6_csv,
+        output_filename=tmp_out_path,
+    )
+    df_trimmer_histogram_from_parquet = pd.read_parquet(tmp_out_path)
+    df_trimmer_histogram_expected = pd.read_parquet(parsed_histogram_LAv5and6_parquet)
+    assert_frame_equal(
```

Comment:
will be nice to modify those assert_frame_equal like we discussed in the future.
Maybe the next time they fail :D

Reply:
Next time they fail sounds good :)